calculatorV2.rb

```diff
@@ -0,0 +1,120 @@
+# Calculator program
+
+# Get operation input from user
+puts "Hey-oh! What kind of calculation would you like to perform?"
+operation = gets.chomp
+
+until operation == "^" || operation == "exp" || operation == "exponent" || operation == "%" || operation == "mod" || operation == "modulo" || operation == "*" || operation == "mult" || operation == "multiply" || operation == "/" || operation == "div" || operation == "divide" || operation == "-" || operation == "sub" || operation == "subtract" || operation == "+" || operation == "add" || operation == "addition"
+	puts "I\'m sorry, I don't understand. Please try again"
+	#kick it back to retrieve new operation and reassign operation
+	operation = gets.chomp
+end 	
+
+
```

Comment:
Super Minor comment:
Just a readability comment. Consider breaking long lines, like line 7 onto multiple lines. The link below may be helpful for looking at that.
http://stackoverflow.com/questions/21417527/how-do-i-write-a-complex-multi-line-if-condition-in-ruby

Other than that though, this chunk of code makes good use of the until block.